easybuild/easyconfigs/r/Racon/Racon-1.4.7-fosscuda-2019b.eb

```diff
@@ -0,0 +1,163 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CMakeMake'
+
+name = 'Racon'
+version = '1.4.7'
+
+homepage = 'https://github.com/clara-genomics/racon-gpu'
+description = "Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/clara-genomics/racon-gpu/archive'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/clara-genomics/ClaraGenomicsAnalysis/archive'],
+        'filename': 'ClaraGenomicsAnalysis.zip',
+        'download_filename': '416af9f1817a4a70745b3f7cdb7418125159f75c.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/bioparser/archive'],
+        'filename': 'bioparser.zip',
+        'download_filename': '0d66534d75b319f43ce5c59fe6548eee48f8f651.zip',
+    },
+    {
+        'source_urls': ['https://github.com/martinsos/edlib/archive'],
+        'filename': 'edlib.zip',
+        'download_filename': 'ee36875e944b63abf6f9b9e14c010b445529427d.zip',
+    },
+    {
+        'source_urls': ['https://github.com/google/googletest/archive'],
+        'filename': 'googletest.zip',
+        'download_filename': 'ea31cb15f0c2ab9f5f5b18e82311eb522989d747.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/logger/archive'],
+        'filename': 'logger.zip',
+        'download_filename': 'b88c8ee8ba59fd07b40c022957b7bdce1a5f8502.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/rampler/archive'],
+        'filename': 'rampler.zip',
+        'download_filename': 'a6dec0140878ba81da90387c77740756fefa4920.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/spoa/archive'],
+        'filename': 'spoa.zip',
+        'download_filename': '11219575bdcb356c57fb32f54cf23b8da2abc4f0.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/thread_pool/archive'],
+        'filename': 'thread_pool.zip',
+        'download_filename': '048da9485390ee8b8a7ecedccc598a06ca4429e2.zip',
+    },
+    {
+        'source_urls': ['https://github.com/madler/zlib/archive'],
+        'filename': 'zlib.zip',
+        'download_filename': 'cacf7f1d4e3d44d871b605da3b647f07d718623f.zip',
+    },
+    {
+        'source_urls': ['https://github.com/google/benchmark/archive'],
+        'filename': 'benchmark.zip',
+        'download_filename': '090faecb454fbd6e6e17a75ef8146acb037118d4.zip',
+    },
+    {
+        'source_urls': ['https://github.com/gabime/spdlog/archive'],
+        'filename': 'spdlog.zip',
+        'download_filename': '366935142780e3754b358542f0fd5ed83793b263.zip',
+    },
+
+]
+checksums = [
+    '7d59c310eb8e7472f6b7bb523206cb0642fe7db32335011cd6b9e7b31cd549ee',  # 1.4.7.tar.gz
+    '228391b8b43effb286cad814ae281c57fad5fcca1724c926cf4cd9ad46b88352',  # ClaraGenomicsAnalysis.zip
```

Comment:
please fix the comments if you rename (but `eb --inject-checksums --force` will do that for you)